main.tf

```diff
@@ -90,7 +92,8 @@ resource "aws_instance" "default" {
     delete_on_termination = "${var.delete_on_termination}"
   }
 
-  tags = "${merge(module.label.tags, map("instance_index", "${count.index}"))}"
+  tags        = "${merge(module.label.tags, map("instance_index", "${count.index}"))}"
+  volume_tags = "${merge(module.label.tags, map("instance_index", "${count.index}"))}"
```

Comment:
looks like `volume_tags` is not in use.
does it needs to be the same as `tags`?

Reply:
nope, isn't a same.  ltmgfy :)

```
volume_tags - (Optional) A mapping of tags to assign to the devices created by the instance at launch time.
```
in case of module it's rood device i.e. `/`

Reply:
@SweetOps 
they are the same in the code, and `volume_tags` is not in use anywhere.
please review it